plugins/chain_api_plugin/chain_api_plugin.cpp

```diff
@@ -181,11 +181,14 @@ namespace {
 
     public:
 
-        resource_calculator(chain::resource_limits_manager& rm, chain::account_name name) :
+        resource_calculator(chain::resource_limits_manager& rm, cyberway::chaindb::chaindb_controller& controller, chain::account_name name) :
           rm_(rm),
+          controller_(controller),
           account_(name),
           prices_(rm.get_pricelist()),
           total_stake_(rm.get_account_stake_ratio(fc::time_point::now(), account_, false).numerator),
+          owned_stake_(controller_.find<eosio::chain::stake_agent_object, eosio::chain::stake_agent_object::by_key>
+                       (eosio::chain::stake::agent_key(chain::symbol(CORE_SYMBOL).to_symbol_code(), account_))->balance),
```

Comment:
I think instead of **->balance** here should be **->get_total_funds()**

Reply:
get_own_funds()

Reply:
here: get_own_funds() - provided